### ISSUE TITLE
Grid toolbar keyboard navigation

### ISSUE THREAD

**Issue by desig9stein** [IgniteUI/igniteui-angular#6040]
## Is your feature request related to a problem? Please describe.
Grid toolbar buttons can't be used with keyboard navigation.

## Describe the solution you'd like
For better accessibility, the user should be able to work with the toolbar using only their keyboard.

## Describe alternatives you've considered
---

## Additional context
---

**Comment by zdrawku** [comment]
This FR is no longer applicable due to the Toolbar refactoring that took place for 10.2.x verions
https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/update-guide#from-102x-to-110x
